server/athenian/api/openapi/openapi.yaml

```diff
@@ -4698,8 +4739,9 @@ components:
       type: object
     FoundJIRAStuff:
       additionalProperties: false
-      description: 'Response from `/filter/jira`: found JIRA epics, labels, and issue
-        types.'
+      description: |-
+        Response from `/filter/jira`: found JIRA epics, labels, issue types, priorities,
+        and mentioned users.
```

Comment:
what are mentioned users here?
When (later) we'll add a table of Jira epics for instance, the filtering should work similarly to how we do it for releases: an epic is an atomic element, we don't break it.

If we select dev X in the filter, epic Y is returned if:
* dev X is assigned to epic Y
* or if dev X is assigned to any tickets (or sub tickets) of epic Y
* or if dev X is the author of any PR linked to that epic (or its children)

Reply:
I think "mentioned" should mean any possible relation of a user to the returned issues, both regular and epics.